x/mongo/driver/operation.go

```diff
@@ -1602,25 +1599,10 @@ func (Operation) canCompress(cmd string) bool {
 // includesHeader: specifies whether or not wm includes the message header
 // Returns the decoded OP_REPLY. If the err field of the returned opReply is non-nil, an error occurred while decoding
 // or validating the response and the other fields are undefined.
-func (Operation) decodeOpReply(wm []byte, includesHeader bool) opReply {
+func (Operation) decodeOpReply(wm []byte) opReply {
```

Comment:
FYI: `OP_MSG`  is supported in MongoDB server version 3.6+: https://github.com/mongodb/specifications/blob/master/source/wireversion-featurelist.rst

I think `OP_REPLY` is only used for the initial command to a server until the wire version is discovered [here](https://github.com/mongodb/mongo-go-driver/blob/14777c12f1e412190bd65750b46b1de155e3f5eb/x/mongo/driver/operation.go#L1055-L1060). The Go driver requires server version 3.6+. I expect decompression for `OP_REPLY` is not commonly executed.

No need to change.